src/dataflow/src/source/timestamp.rs

```diff
@@ -71,17 +71,21 @@ impl TimestampProposer {
         }
     }
 
-    /// Attempt to propose that `(partition, offset)` be bound to `time`, which means
-    /// that all offsets < `offset` get bound to `time` for `partition`.
+    /// Propose that `(partition, offset)` be bound to whatever timestamp we are
+    /// currently assigning data to.
```

Comment:
This might be more clearly stated as `self.timestamp`?

Reply:
done